addon/pods/components/frost-tab/template.hbs

```diff
@@ -1 +1,15 @@
-{{yield}}
+<div class='tab' data-test={{hook (concat hook '-tab' (if isSelected '-selected' '')) }}>
```

Comment:
I wouldn't concat the hook, look into making selected a qualifier https://www.npmjs.com/package/ember-hook#qualifiers